examples/bridge-app/linux/main.cpp

```diff
@@ -319,6 +305,7 @@ int RemoveDeviceEndpoint(Device * dev)
     {
         if (gDevices[index] == dev)
         {
+            DeviceLayer::StackLock lock;
```

Comment:
Again, not going to work on Mac...

Reply:
Added comment to indicate the change to schedule the work should be made.